src/components/AccountSelector/AccountSelectButton/AccountSelector.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { FiUser } from "react-icons/fi";
+import { MassaToken } from "../../../assets/svg-components/MassaToken";
+import { AccountSelectorButton } from "./AccountSelectorButton";
+
+export default {
+  title: "Components/Account Selector",
+  component: AccountSelectorButton,
+};
+
+const theme = "theme-light";
+
+const args = {
+  profileimage: <FiUser className="text-neutral h-6 w-6" />,
+  accountname: "account #",
+  svg: <MassaToken size={"24"} />,
+  accountbalance: "0,000.00",
```

Comment:
**profileimage**, please camelCase: **profileImage**. Why not Avatar or Image simply?

**accountname** camelCase once again.

Svg is not a good param name. Let's think in something generic for the picto there. BTW, **size** should not be **string** but **integer**.

What if the **accountbalance**  value is not balance linked!? 🤔 Use something more generic like: **amount**

Reply:
Done !